src/lib/comments.ts

```diff
@@ -41,6 +44,39 @@ export const buildThreads = (
     }))
 }
 
+/**
+ *  will filter out orphan comments with their reply nodes from comments plugin,
```

Comment:
The comment is a bit misleading, it says `filter out orphan comments from comments plugin` but actually the comments coming from the plugin already filtered and does not include orphan comments.

Reply:
updated